.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Yeah, beta is being suggested only for 3 days now: https://github.com/ream/ream#install